atom/browser/api/atom_api_web_contents.cc

```diff
@@ -890,6 +890,26 @@ void WebContents::DidFinishNavigation(
   }
 }
 
+void WebContents::SecurityStyleChanged(
+    content::SecurityStyle security_style,
+    const content::SecurityStyleExplanations& explanations) {
+    std::string type = "unknown";
```

Comment:
merging but see `template<> struct Converter<atom::api::WebContents::Type>` at the top of this file for converting between C++ types and JS for node apis